### ISSUE TITLE
Do not remember some tool options

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#980]
I think that many tool settings should NOT be remembered.

For example, here is a very frequent mistake of users from my experience:
User sets a tool blending mode from "Composite" (default) to "Onto" (accidentally or without knowledge of its meaning).
As result, tool draws nothing.
User restarts application, but Blend Method is remembered, so nothing changes.
Then angry user records a video about "bad application that doesn't work".
(Real story, happened more than once).

Another example - #964.

So, I think some tool settings should not be remembered.

Question: which ones? We have to define a list.

**Comment by ebarranco** [comment]
HI, I think with an option in the settings to check if we want preserve our values or not is enough and by default Synfig shouldn't do it.

**Comment by Jose-Moreno** [comment]
I agree with Enrique that having a toggle in the preferences that remembers or not the modified values, as well as being turned off by default seems like a proper solution. That way we would always have the Synfig default values on every new file, and if someone wants to keep the same values between scenes in a larger project, for example, they can enable that option instead.

To complement the latter scenario, it could also make sense to make a "template" file perhaps from a copy of the previous scene, and work over that, so the values could ne stored per file and not globally.

I'm not sure how often does a single user actually need to use the same specific opacity setting for a shape layer or having the same line width between files (unless one is working on a larger production team), but depending on workflow this could be a proper option for users that build rigs or prepare the scenes for animators to keep consistency on tool settings for the whole show / production.

**Comment by morevnaproject** [comment]
> Hi, I think with an option in the settings to check if we want preserve our values or not is enough and by default Synfig shouldn't do it.

How about a button to save tool settings?
By default Synfig will not save settings. But if user wants to save his current configuration of a tool, then he clicks this button and his current settings remembered.

**Comment by ebarranco** [comment]
I'm not sure if I understand how a button will work. How about when I don't want to that happen anymore, You have to press it again? Where this button will be? I don't know, it makes better sense for me have a check-mark on the Preferences->System, saying something like "Preserve my tool options values" or something like that. Do it once and  your ready to go.

**Comment by morevnaproject** [comment]
Let me explain how I see this:

1.  The application **never** saves tool options on quit.

2. The button is expected to be in tool options dialog
![screenshot_001](https://user-images.githubusercontent.com/332868/68068017-7ad70b00-fd82-11e9-876f-93225e7107a2.png)

3. When user clicks the button it saves current options for current tool. The save happens immediately when user clicked button. So, when Synfig starts next time it will restore tool settings which were configured when user clicked this save button last time.

4. @blackwarthog suggest also to add "reset tool settings to defaults" button. It can be placed right near the save button.

**Comment by blackwarthog** [comment]
> 4. @blackwarthog suggest also to add "reset tool settings to defaults" button. It can be placed right near the save button.

Actually I did not suggest to "also" add button. I've suggested to add button *instead* of removing existing functintality. I'm for keep feature of saving of tool state and think it is a bad idea to remove this. That you just forgot about this makes me sad.

I suggest to add the "reset tool settings" button at the conspicuous place, near current color for example, and keep the feature with saving of tool state.

**Comment by ice0** [comment]
I am also agree with Enrique that the current implementation is wrong and needs to be changed.
Intuitive behavior - forget all changes after exit. And switch to the old behavior only if the user wants it.
